spec/game_spec.rb

```diff
@@ -1,17 +1,19 @@
 require 'game'
 
-describe Game do
-  let(:game) { Game.new }
+module RubyConnect
+  describe Game do
+    let(:game) { subject }
 
-  it 'has an empty board' do
-    expect(game.board).to have_attributes(size: 7)
-  end
+    it 'has an empty board' do
```

Comment:
We are also testing for empty board on the board specs, should be delete one of the test?